docroot/themes/humsci/humsci_basic/src/scss/components/_main-nav.scss

```diff
@@ -1,105 +1,354 @@
+    }
+
+    .hb-main-nav__toggle-icon {
+      @include hb-menu-hamburger-icon;
+      margin-left: hb-calculate-rems(10px);
     }
 
-    // Close Icon
-    &[aria-expanded=true]::after {
-      background-image: svg('<svg viewBox="0 0 24 24"><path d="M10.56,12,.32,22.21a1,1,0,0,0,1.41,1.41L12,13.39,22.21,23.62a1,1,0,0,0,1.41-1.41L13.39,12,23.62,1.74a1,1,0,0,0,0-1.42,1,1,0,0,0-1.41,0L12,10.56,1.74.32A1,1,0,0,0,.32,1.74Z" fill="#2e2d29" transform="translate(-0.03 -0.03)"/></svg>') !important;
+    &[aria-expanded="true"] .hb-main-nav__toggle-icon {
+      @include hb-menu-close-icon;
     }
   }
 
   &__menu {
+    // scss-lint:disable ImportantRule
     box-shadow: none !important;
     transform: translate(0) !important;
     width: auto !important;
+    // scss-lint:enable ImportantRule
 
     list-style-type: none;
     margin: 0;
     padding: 0;
 
-    line-height: 2;
-
     @include grid-media-min('sm') {
       max-width: none;
     }
 
+    &[aria-hidden="true"] {
+      display: none;
+    }
+
     &-lv1 {
-      position: relative !important;
+      font-weight: hb-theme-font-weight(bold);
+      font-size: hb-calculate-rems(22px);
+      border-bottom: $hb-menu-light-border;
 
-      &[aria-hidden="true"] {
-        display: none !important;
+      &[aria-hidden="false"] {
+        margin-bottom: hb-calculate-rems(28px);
       }
 
-      @include grid-media('md') {
-        display: flex;
-        max-width: none !important;
+      @include grid-media-min('md') {
+        font-size: hb-calculate-rems(30px);
       }
 
-      @include grid-media('lg') {
-        .su-main-nav__menu-container {
-          position: absolute;
+      @include grid-media-min('lg') {
+        .js & {
+          font-size: hb-calculate-rems(18px);
+          font-weight: hb-theme-font-weight(semibold);
+          display: flex;
+          align-items: flex-end;
+          border: 0;
+
+          &[aria-hidden="false"] {
+            margin: 0;
+          }
         }
       }
     }
 
     &-lv2 {
-      @include grid-media('lg') {
-        // Hide any nested menus that are level 3 or greater
-        .su-main-nav__button,
-        .su-main-nav__menu-container {
-          display: none;
+      font-weight: hb-theme-font-weight(regular);
+      font-size: hb-calculate-rems(18px);
+
+      @include grid-media-min('md') {
+        font-size: hb-calculate-rems(22px);
+      }
+
+      @include grid-media-min('lg') {
+        .js & {
+          font-weight: hb-theme-font-weight(semibold);
+          font-size: inherit;
         }
       }
     }
-  }
 
-  &__menu-container {
-    padding-left: hb-calculate-rems(20px);
-    margin-top: hb-calculate-rems(8px);
-    margin-bottom: hb-calculate-rems(8px);
-
-    &[aria-hidden="true"] {
-      display: none;
+    &-lv3 {
+      @include grid-media-min('lg') {
+        .js & {
+          display: none;
+        }
+      }
     }
   }
 
   &__item {
     position: relative;
+
+    .hb-main-nav__menu-lv1 > & {
```

Comment:
the two styles here are almost identical, maybe you can do a selector like:

```
.hb-main-nav__menu-lv1 > &,
.hb-main-nav__menu-lv2 &:not(:last-child) {
      @include grid-media-min('lg') {
        .js & {
          border: 0;
        }
      }
}


```

Reply:
That complicates things a bit since I still need to reference those selectors to add margin and border styles. It's a bit cleaner to keep things separated.